### PR TITLE
Docket Switch | Edit Tasks w/o Existing Tasks

### PR DESCRIPTION
**Pull request by jcq**
### Description
Minor tweaks to copy displayed on Docket Switch Add/Remove Tasks view, per @rutvigupta-design 

### Acceptance Criteria
- [ ] Code compiles correctly
- [ ] Copy changes, per updated design

### Updated Design

DS Updated Task View

### CODE REVIEW

**Comment by jcq** [comment]
## Screen Shot

Screen Shot 2021-02-04 at 16 12 44

**Comment by jcq** [comment]
@rutvigupta-design What margin did you want between the "Please unselect..." and the italic text?

**Comment by rutvigupta-design** [comment]
@jcq let's do 16px between the "Please unselect..." and the italic text below, if possible!

File: client/app/queue/docketSwitch/grant/DocketSwitchEditTasksForm.jsx [hunk 1]
```diff
@@ -169,7 +169,7 @@ export const DocketSwitchEditTasksForm = ({
             />
           </div>
 
-          {taskOptions?.length && (
+          {/* {Boolean(taskOptions?.length) && (
```
Comment (nanotone) [inline client/app/queue/docketSwitch/grant/DocketSwitchEditTasksForm.jsx:172]: This chunk of commented-out code is mirrored in the new code below. What's the benefit of keeping this around?

**Review by nanotone — APPROVED** [review]
Non-blocking question about commenting out code, but LGTM.

File: client/app/queue/docketSwitch/grant/DocketSwitchEditTasksForm.jsx (hunk 1 above)
Comment (jcq) [reply client/app/queue/docketSwitch/grant/DocketSwitchEditTasksForm.jsx]: Whoops, I thought I had removed that. Good catch!

**Pull request merged**